packages/core/src/loaders/filesystem/__tests__/filesystemLoader.unit.ts

```diff
@@ -21,7 +21,7 @@ describe('filesystemLoader', () => {
 
   test('given no opts should delegate to graph facade for http operations', async () => {
     jest.spyOn(fs, 'existsSync').mockReturnValueOnce(true);
-    jest.spyOn(fs, 'statSync').mockReturnValueOnce({ isDirectory: () => false });
+    jest.spyOn(fs, 'statSync').mockReturnValueOnce({ isDirectory: () => false } as fs.Stats);
```

Comment:
This needs to disappear but my mind is gone right now and I can't think of any possible solution. It's clear that we're mocking too much stuff and the API for this FileSystem Reader should probably be changed.

Reply:
@XVincentX it's a unit test and it should mock all dependencies, why do you think it's mocking too much?

Reply:
You're using a single property from an entire gigantic object; this flag could be passed somehow as a parameter so we can mock that thing in particular and not the whole `statSync` returned object.

Reply:
:+1: makes sense